parallaxinc/Parallax-IDE
# adds confirm dialog on overwrite

#### What's this PR do?

adds a confirmation overlay when attempting to overwrite an existing file
#### Where should the reviewer start?

the logic change starts in the handlers.js
#### How should this be manually tested?

pull down, build, load, then create a new file.  Attempt to save it with the name of an existing file.
Also, save a new file with a name that doesn't exist yet and dialog should not appear.
#### What are the relevant tickets?

fixes #168
